src/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -62,9 +60,6 @@ public class SslStreamNetworkStreamTest
 
                     await Task.WhenAll(clientAuthenticationTask, serverAuthenticationTask);
 
-                    byte[] readBuffer = new byte[256];
-                    Task<int> readTask = clientStream.ReadAsync(readBuffer, 0, readBuffer.Length);
```

Comment:
I'm not clear on why this is being removed.  The mistake here doesn't look like doing the read but rather doing the read but then not awaiting the resulting task... we should instead add the readTask to the WaitAll below.